.gitignore

```diff
@@ -25,6 +25,7 @@ target/
 *~
 
 .gradle
+.vscode/
```

Comment:
don't ignore .vscode

but ignore .mvn/wrapper/maven-wrapper.jar

Reply:
the maven-wrapper.jar will be downloaded automatically if missing from the directory, so no need to pollute the git repo with that binary